Client/Frontend/photon-colors.swift

```diff
@@ -80,6 +82,7 @@ extension UIColor {
         static let Grey75 = UIColor(rgb: 0x3C3C43)
         static let Grey75A60 = UIColor(rgba: 0x3C3C4399)
         static let Grey80 = UIColor(rgb: 0x2a2a2e)
+        static let Grey85 = UIColor(rgb: 0x1C1C1e) // system background / dark / secondary
```

Comment:
is there a way we can just use the system colour? Or do we have to define it ourselves

Reply:
We can use the system colours in iOS13+ and have them dynamically adjust based on the system theme, but not below. I was upset by this. This is how we found out that we support iOS 11.4. hahahaa. But also sadface.